### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by stephend017**
Added support for validating on the frontend

### CODE REVIEW

**Review by sotails — APPROVED** [review]

File: biit_server/azure.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from typing import Tuple
+
+CLIENT_ID = "c128fe76-dc54-4daa-993c-1a13c1e82080"
+TENANT_ID = "4130bd39-7c53-419c-b1e5-8758d6d63f21"
+REDIRECT_URI = "https://auth.expo.io/@stephend17/biit-mobile"
+
+def azure_refresh_token(refresh_token: str) -> Tuple[str, str]:
+    """
+    Refreshes a given refresh token and returns the
+    new access token and refresh token
```
Comment (fatcat2) [inline biit_server/azure.py:11]: Add params in docstring.

File: biit_server/azure.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    payload = f'client_id={CLIENT_ID}&scope=https://graph.microsoft.com/User.Read&redirect_uri={REDIRECT_URI}&grant_type=refresh_token&refresh_token={refresh_token}'
+    headers = {
+        'Content-Type': 'application/x-www-form-urlencoded',
+    }
+
+    response = requests.request("POST", url, headers=headers, data=payload)
+    rjson = response.json()
+
+    if response.status_code != 200:
+        return ("", "")
```
Comment (fatcat2) [inline biit_server/azure.py:24]: Should we return an error here?

**Review by fatcat2 — APPROVED** [review]
Make the small changes and we are good! to! go!

File: biit_server/azure.py (hunk 2 above)
Comment (stephend017) [reply biit_server/azure.py:24]: Probably but I don't know what error to return here. I think we should discuss this at our next meeting.

**Review by fatcat2 — APPROVED** [review]
looks! good! to! me!

**Pull request merged**